### DISCUSSION TITLE
Hard to find smd IC's

### DISCUSSION THREAD

**Discussion by retroshire** [frntc/Sidekick64#11] [Q&A]
I tried to find the materials for building the Sidekick64.

74HCT30 is almost unfindable in SO:SOIC package, on Aliexpress there a SOP package. Is this a suitable piece?

Is the 74HC30 an alternative, it has no TTL.

And the other packages, are they exclusive?

**Comment by frntc** [comment]
Hi,

we already had mail exchange, but since this answer might be of interest for others as well: you can instead use a 74LS30. This IC is running "in" the 5V circuitry of the C64. Using HC is generally not recommended and might cause problems. All other ICs are running on 3.3V but need 5V-tolerant pins.

**Comment by banman11** [comment]
Go to this site.....

https://www.nexperia.com/

I got all the hard to find parts at great prices. All original manufacturer as well.

Good luck!

I hope this helps.....

**Comment by banman11** [comment]
https://www.nexperia.com/search.html?q=74HCT30&searchCategory=xref


Look for SO14. I think this is the correct package size we need. 

Please correct me if I am wrong.

**Comment by frntc** [comment]
Yes, SO14 is correct, either LS30 or HCT30 (avoid others, e.g. HC30 will very likely not work at all)

**Reply by banman11** [reply]
Absolutely agree. Unpredictable logic states will undoubtedly manifest themselves. 

https://www.allaboutcircuits.com/textbook/digital/chpt-3/logic-signal-voltage-levels/

**Comment by banman11** [comment]
I have built a working version of sidekick64.
All IC components as prescribed in BOM appear to be operating (even the suspect ones from AliExpress).
I highly recommend getting quality parts from respected manufacturer/distributers as mentioned above.
I have only used the loading of disk based demo programs so far.
The loading speed is instantaneous.
